lib/oauth2/client.rb

```diff
@@ -39,7 +40,8 @@ def initialize(client_id, client_secret, options = {}, &block)
                   :connection_opts  => {},
                   :connection_build => block,
                   :max_redirects    => 5,
-                  :raise_errors     => true}.merge(opts)
+                  :raise_errors     => true,
+                  :logger           => ::Logger.new($stdout)}.merge!(opts)
```

Comment:
There is one small, unrelated change happening here that may be difficult to spot:

```
# before
opts = {}.merge(opts)

# after, #merge! modifies original hash rather than instantiating another
opts = {}.merge!(opts)
```